services/galley/src/Galley/API/MLS/Removal.hs

```diff
@@ -170,6 +196,10 @@ removeClient lc qusr c = do
     let getClients = fmap (cid,) . cmLookupIndex cid . (.members)
     removeClientsWithClientMapRecursively (qualifyAs lc mlsConv) getClients qusr
 
+data RemoveUserIncludeMain
```

Comment:
Could we have some comments explaining what this means?

Reply:
Added some docs. Hopefully that clarifies it.